components/should_verify_email/should_verify_email.tsx

```diff
@@ -71,7 +84,7 @@ export default class ShouldVerifyEmail extends React.PureComponent {
                             {(title) => (
```

Comment:
Can we move the type declaration here?
```suggestion
                            {(title: string) => (
```